graph/resolvers/root_mutation.js

```diff
@@ -48,7 +48,13 @@ const RootMutation = {
   },
   setCommentStatus(_, {id, status}, {mutators: {Comment}}) {
     return wrapResponse(null)(Comment.setCommentStatus({id, status}));
-  }
+  },
+  addCommentTag(_, {id, tag}, {mutators: {Comment}}) {
+    return wrapResponse('comment')(Comment.addCommentTag({id, tag}));
+  },
+  removeCommentTag(_, {id, tag}, {mutators: {Comment}}) {
+    return wrapResponse('comment')(Comment.removeCommentTag({id, tag}));
```

Comment:
As with above, we shouldn't have this return a "comment", rather null as above when performing a mutation on the comment itself.